### ISSUE TITLE
Option to hide / remove cloud based calendars?

### ISSUE THREAD

**Issue by rafnov** [Etar-Group/Etar-Calendar#1717]
I have multiple google accounts on my device. I try not use them at all. In Google calendar I just need to change main to local, when adding event.
Etar shows all of them in options.
Would be nice to hide / disable them unused accounts by just a simple toggle.

**Comment by JohnVeness** [comment]
You can choose which calendars to display in Etar's settings.

**Comment by JustSavil** [comment]
Yeah it is tiring to do the cloud based calendars one by one. I wanted to have a completely seperate local home server only calendar from the cloud calendar from google. When I change the settings on Etar, unfortunately those settings carry over to the android calendar app and vice versa. How can I stop thar from happenening? Putting Etar in Android secure folder?

**Comment by vdbhb59** [comment]
Good point. Anyone got around to it yet?
